libafl_bolts/src/llmp.rs

```diff
@@ -808,6 +803,22 @@ pub struct LlmpPage {
     pub messages: [LlmpMsg; 0],
 }
 
+impl LlmpPage {
+    #[inline]
+    fn receiver_joined(&mut self) {
+        let receivers_joined_count = &mut self.receivers_joined_count;
+        //receivers_joined_count.fetch_add(1, Ordering::Relaxed);
+        receivers_joined_count.store(1, Ordering::Relaxed);
+    }
+
+    #[inline]
+    fn receiver_left(&mut self) {
+        let receivers_joined_count = &mut self.receivers_joined_count;
+        //receivers_joined_count.fetch_sub(1, Ordering::Relaxed);
+        receivers_joined_count.store(0, Ordering::Relaxed);
```

Comment:
I still don't really get this one

shouldn't this be
receivers_left_count += 1 or something?
I don't see receivers_left_count is incremented anywhere

Reply:
Yes, good point

Reply:
Fixed, thanks!

Reply:
"receivers_joined_count and receivers_left_count are both either 0 or 1."
is this correct?